### PR TITLE
change device_tracker icon

### PR DESCRIPTION
**Pull request by Mariusthvdb**
from fixed hass:account to state dependent lan-connect/lan-disconnect

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/common/entity/domain_icon.ts [hunk 1]
```diff
@@ -88,6 +87,9 @@ export const domainIcon = (domain: string, state?: string): string => {
           return "hass:window-open";
       }
 
+    case "device_tracker":
+      return state && state === "home" ? "hass:lan-connect" : "hass:lan-disconnect";
```
Comment (MartinHjelmare) [inline src/common/entity/domain_icon.ts:91]: Should we check the state attribute `"source_type"` too and only use these icons if that attribute is `"router"`. There are different categories of device trackers. Eg GPS trackers should perhaps not be represented with the lan icons?

**Comment by MartinHjelmare** [comment]
Side note: I think you're targeting the wrong branch in this PR. Normally we target the dev branch.

File: src/common/entity/domain_icon.ts (hunk 1 above)
Comment (Mariusthvdb) [reply src/common/entity/domain_icon.ts:91]: ok, thats a good idea, thx for the suggestion

Comment (MartinHjelmare) [reply src/common/entity/domain_icon.ts:91]: I'm not a frontend expert. Someone in the frontend team should continue the review. 👍

Comment (ludeeus) [reply src/common/entity/domain_icon.ts:91]: Your branch is old, this file does not look anything like this anymore.
If you rebase (or start over) with the current dev, how to do this based on attributes should be clearer. (Look at cover for instance)

**Review by ludeeus — CHANGES_REQUESTED** [review]
- Needs rebase
- Target needs to be against `dev`
- The suggested icons should only apply to the "router" "source_type" (and default back to the current icon)

**Comment by Mariusthvdb** [comment]
Thanks, I'll start over... please show me which file I need to PR?

**Comment by ludeeus** [comment]
The same file, just need something that is up to date.
<https://github.com/home-assistant/frontend/blob/dev/src/common/entity/domain_icon.ts> This is how the file look in `dev`

**Comment by Mariusthvdb** [comment]
thanks, closing this.

**Pull request closed without merging**